lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -334,12 +371,10 @@ if Enum.all?(
 
     defp get_hls_sink_spec(state, track, directory) do
       hls_sink = %Membrane.HTTPAdaptiveStream.SinkBin{
-        manifest_module: Membrane.HTTPAdaptiveStream.HLS,
+        manifest_module: state.manifest_module,
         target_window_duration: state.target_window_duration,
-        persist?: false,
```

Comment:
Why was this flag previously set to false?

Reply:
when `persist` is set to true, in case of terminating a pipeline we won't lose any data. Previously we were able to persist data only when we closed event with end button.